api/src/main/java/marquez/db/RunDao.java

```diff
@@ -190,9 +192,10 @@ SELECT run_uuid, JSON_AGG(json_build_object('namespace', namespace_name,
           + "nominal_start_time = COALESCE(EXCLUDED.nominal_start_time, runs.nominal_start_time), "
           + "nominal_end_time = COALESCE(EXCLUDED.nominal_end_time, runs.nominal_end_time), "
           + "location = EXCLUDED.location "
-          + "RETURNING *")
-  ExtendedRunRow upsert(
+          + "RETURNING uuid")
```

Comment:
Do we want to change the contract here to return `uuid` instead of `ExtendedRunRow`?